<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1426</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1461</width>
      <height>831</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>222</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="patientNameLabel">
                 <property name="text">
                  <string>Patient Name:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="patientName"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="emailNameLabel">
                 <property name="text">
                  <string>Email:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="email"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="phoneNumberLabel">
                 <property name="text">
                  <string>Phone-Number:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="phone"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="OHIPLabel">
                 <property name="text">
                  <string>OHIP: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="ohip"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="bdayLabel">
               <property name="text">
                <string>Birth day</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="birthDay"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="sexLabel">
             <property name="text">
              <string>Gender</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="gender">
             <item>
              <property name="text">
               <string>Male</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Female</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Non-Binary</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Trans-Man</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Trans-Woman</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Other</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="newPatientButton">
         <property name="text">
          <string>New Patient</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="recordstextBrowser">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>1121</width>
        <height>781</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>262</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>New Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>New Email:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="newEmail"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>New Phone-Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="newPhone"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>New Gender</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="newGenderComboBox">
           <item>
            <property name="text">
             <string>Male</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Female</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Non-Binary</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Trans-Man</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Trans-Woman</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Other</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>262</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Patient To Update:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11"/>
       </item>
       <item>
        <widget class="QLineEdit" name="patientToUpdate"/>
       </item>
      </layout>
     </widget>
     <widget class="QTextBrowser" name="updateDisplay">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>1101</width>
        <height>771</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="updateButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>311</width>
        <height>318</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="searchedPatientLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Time of Appointment:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>-140</x>
                <y>-30</y>
                <width>313</width>
                <height>320</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8"/>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="appointDate"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="checkAppointmentsButton">
         <property name="text">
          <string>Check Appointments</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Time of Appointment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="appointTime"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Summary:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="reasonForAppointment"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>344</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Search Patients</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchPatient"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="searchPatientButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="makeAppointmentButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>380</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Make Appointment</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="appointmentsDisplay">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>1021</width>
        <height>761</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1426</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
